bedrock/settings/base.py

```diff
@@ -950,6 +950,11 @@ def lazy_email_backend():
 FXA_IFRAME_SRC = config('FXA_IFRAME_SRC',
                         default='https://accounts.firefox.com/')
 
+# Bug 1264843: embed alternative FxA servers within partner distributions
+FXA_IFRAME_SRC_DIST_MAP = config('FXA_IFRAME_SRC_DIST_MAP',
```

Comment:
How likely do we think it is we'll get multiple partner distributions? This seems robust, but then I wonder if it's maybe overkill. Happy to defer to what you think here.